server.js

```diff
@@ -1,11 +1,15 @@
 const express = require('express');
-const graphqlHTTP = require('express-graphql');
-const app = express();
+const { graphqlHTTP } = require('express-graphql');
```

Comment:
We destructure this to get the graphqlHTTP function because graphqlHTTP is a child of the express-graphql import. See [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) for more info.